query/src/interpreters/interpreter_copy.rs

```diff
@@ -111,7 +112,7 @@ impl CopyInterpreter {
     //  COPY should deal with the file one by one and do some error handler on the OnError strategy.
 
     #[tracing::instrument(level = "debug", name = "copy_one_file_to_table", skip(self), fields(ctx.id = self.ctx.get_id().as_str()))]
-    async fn copy_one_file_to_table(&self, file_name: Option<String>) -> Result<()> {
+    async fn copy_one_file_to_table(&self, file_name: Option<String>) -> Result<Vec<DataBlock>> {
```

Comment:
Not sure that if a file is 100GB if we return the Vec<DataBlock> is it OOM?

Reply:
It's the append entry result(with metadata: append id, ..), not the actual data block.